src/material-experimental/mdc-progress-spinner/progress-spinner.scss

```diff
@@ -0,0 +1,23 @@
+@import '@material/circular-progress/mixins.import';
+@import '../mdc-helpers/mdc-helpers';
+
+
+@include mdc-circular-progress-core-styles($query: $mat-base-styles-without-animation-query);
+
+.mdc-circular-progress {
```

Comment:
Usually we have our our `mat-mdc-` prefixed classes instead of targeting the MDC classes directly so that the styles don't conflict with something else on the page.

Reply:
Just to further clarify, it's fine for `.mdc-*` classes to appear somewhere in the selector, as long as the selector as a whole is specific to our usage of it, e.g. below you have: `.mat-mdc-progress-spinner .mdc-circular-progress--indeterminate {`. This is ok since its guarded by a `.mat-mdc-*` class